.github/workflows/package-ci.yml

```diff
@@ -21,10 +21,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os:
-          - ubuntu-22.04
-          - macos-10.15
-          - windows-2022
+        include:
+          - name: 🐧 Linux
+            os: ubuntu-22.04
+          - name: 🍎 macOS
+            os: macos-10.15
+          - name: 🏁 Windows
+            os: windows-2022
+    name: '${{ matrix.name }}: 📦 Packaging'
```

Comment:
```suggestion
    name: '${{ matrix.name }}: 📦 Packaging Wheel'
```
?
```suggestion
    name: '${{ matrix.name }}: 📦 Packaging 🛞'
```
?

Reply:
Good catch ! I couldn't display you second suggestion (must be an exotic emoji :/) so I stick with a simple wording